Modules/mathmodule.c

```diff
@@ -53,6 +53,7 @@ raised for division by zero and mod by zero.
  */
 
 #include "Python.h"
+#include "pycore_bitutils.h"
 #include "pycore_dtoa.h"
 #include "_math.h"
 
```

Comment:
Maybe add an helper function which takes a digit, to make the code more future proof if tomorrow a digit becomes larger than unsigned long. Something similar to:

```
static int
popcount_digit(digit d)
{
    // digit can be larger than uint32_t, but only PyLong_SHIFT bits
    // of it will be ever used.
    Py_BUILD_ASSERT(PyLong_SHIFT <= 32);
    return _Py_popcount32((uint32_t)d);
}
```

Also because digit is signed, whereas _Py_bit_length() argument is unsigned.

Reply:
Sorry, this comment is for longobject.c.